src/player.cpp

```diff
@@ -10483,11 +10483,19 @@ bool player::can_wield( const item &it, bool alert ) const
             if( alert ) {
                 add_msg( m_info, _("The %s can't be wielded with only one arm."), it.tname().c_str() );
             }
+            if( worn_with_flag("RESTRICT_HANDS") ) {
+                add_msg( m_info, _("Something you are wearing hinders the use of both hands."),
```

Comment:
Problem is the wrong item will be cited. To quote @Coolthulhu:

> it refers to the item you're trying to wield, not the item that restricts wielding.
For example, if you tried to wield a battle axe while already wearing a briefcase, you'd get a message "The battle axe you are wearing...".